### ISSUE TITLE
hub: dock creation with weird characters in ID 

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#1751]
This happened this weekend too and I thought it was a fluke but it’s now happened again so I tell you. In creating a new dock with ID logsipad and name Luiza’s iPad I get the following automatically, with weird characters. If I go in and edit the name back to Luiza’s iPad I am able to save it as such, but it presumably shouldn’t add all of that weird stuff in the first place.

![65164137-6CAA-4819-93D7-B108FE69A9FA](https://user-images.githubusercontent.com/67970402/178376953-93c7737d-bf86-4246-8b87-17b3cc7a4aa0.png)

![85F0D0A8-78BE-4DFD-A40C-65BA6CAEB161](https://user-images.githubusercontent.com/67970402/178376972-55888719-c135-40db-b035-51c15fc7bb3a.png)

**Comment by luizaogs** [comment]
This was on my iPad.

**Comment by urapadmin** [comment]
that's so weird. So if you go in and save it again it saves it correctly?

**Comment by urapadmin** [comment]
you sure?

**Comment by luizaogs** [comment]
That it saves correctly after I edit it? I mean, I’m using Luiza’s iPad in PACHA now :) Or at least I was yesterday.

**Comment by urapadmin** [comment]
I just thought I'd play the sure? yes? really? of course? game with you. Because what else can I do without a clue?

**Comment by luizaogs** [comment]
Watch this be another weird Luiza bug that no one else can reproduce. I have a talent, what can I say?

**Comment by urapadmin** [comment]
some Silicon Valley companies pay a lot of money for that talent. Good you have other talents, too :o)

**Comment by urapadmin** [comment]
I take it your iPad is not set to Portuguese (is that the right spelling for the language? Or is it only for people?), right?

**Comment by luizaogs** [comment]
Nope, to English. 

(that is the right spelling for the language!)

**Comment by urapadmin** [comment]
and you did use the one not french thingy, the single quotation mark, right? (Now I am playing the asking silly questions game. Next I will ask you if the power plug is in)

**Comment by luizaogs** [comment]
You might have to get a technician to come check that one out, I’m not sure

**Comment by luizaogs** [comment]
(yes, the single quotation mark :))

**Comment by urapadmin** [comment]
funny, just occurred to my Riesling-infused brain for the first time that there might be a difference between what the language is called and what the people are called. Apart from the fact, of course, that some cultures use some other language. Now my brain tries to find an example where that differs. (English and the Brits does not count)

**Comment by urapadmin** [comment]
Browser?

**Comment by luizaogs** [comment]
You mean like Brazilian and Portuguese? I do get often asked if I speak Brazilian. Always fun.

**Comment by luizaogs** [comment]
Safari, which is probably an answer you will both expect and dislike

**Comment by urapadmin** [comment]
no, that is like Canadians and French or Canadians and English. There is no Canadian as a language. Debatable with the Australians, but that's a different story. I guess India is an example. There is no Indian. But there are Indians.

**Comment by urapadmin** [comment]
can you please try that with Chrome. Just for fun.

**Comment by urapadmin** [comment]
ja. Riesling-no-brainer. There are Egyptians but there is no Egyptian. Anymore.

**Comment by luizaogs** [comment]
Ha ha ha I was about to mention the Egyptian case (though many Egyptians would argue that their Arabic is indeed Egyptian, I think). And yes, was already grabbing my iPad…

**Comment by luizaogs** [comment]
Alas

![E3726292-810D-4DDD-B99C-F26970088ED5](https://user-images.githubusercontent.com/67970402/178600262-4d37c61b-599b-411f-9dae-b630fc7f9383.jpeg)

**Comment by urapadmin** [comment]
that does not look right

**Comment by urapadmin** [comment]
what do I do if I can't blame it on Safari?

**Comment by urapadmin** [comment]
my last line of defense

**Comment by luizaogs** [comment]
…blame it on Apple. That one seems to always work.

**Comment by urapadmin** [comment]
I have to solve it, notwithstanding (a rather German English word). 
Just don't go work for Silicon Valley.

**Comment by luizaogs** [comment]
But that means that I have to finish my dissertation…

**Comment by urapadmin** [comment]
The world is waiting for it!

**Comment by urapadmin** [comment]
(no pressure, here)

**Comment by luizaogs** [comment]
👀

**Comment by urapadmin** [comment]
I also see all these other ... erm ... students get PhDs, so I am afraid, I am really relying on you. Somebody has to earn it.

**Comment by lbestock** [comment]
Well now I know what I will do with my evening. Having read much unsatisfying scholarship today I will read Chapter 2 while I cook dinner.

**Comment by urapadmin** [comment]
still, no pressure ...

**Comment by urapadmin** [comment]
I like it when we get off topic on a bug I am clueless about :o)

**Comment by lbestock** [comment]
Yeah, well, Luiza last time I checked you were actually aware of the difference between observations and interpretations. And also, as to language names and what not, I read the most egregious pots are people I have encountered in years today. Egyptian pottery to Egyptians in lightspeed, Sheikh Mufta ceramic complex to Sheikh Muftians faster than you can blink.

**Comment by luizaogs** [comment]
Always my tickets. Woops. Another talent of mine…

**Comment by luizaogs** [comment]
Well yes if your bar is that low then I’ll be able to swing a bit above it!

**Comment by urapadmin** [comment]
I leave this trail. The most egregious pots are people?

**Comment by luizaogs** [comment]
Archaeology speak. Pots = people, so if you find an Egyptian pot in ancient China the author of the chapter Laurel read would probably say that it was an Egyptian migration to China.

**Comment by lbestock** [comment]
"Pots aren't people" is an old archaeological adage that despite being a classic gets violated all the time. You find a Nubian pot, not a Nubian person. And "Nubian" as defining pottery is an archaeological category anyway, we have no idea what its actual relationship is to people who felt themselves to be a group at all, let alone a Nubian group. But you'd be amazed how often people find Nubian pots and leap to assuming they are looking at Nubians.

**Comment by urapadmin** [comment]
aha

**Comment by urapadmin** [comment]
sure

**Comment by urapadmin** [comment]
🪴 only potted plants, no :pot

**Comment by lbestock** [comment]
Is that a potted pot plant?

**Comment by lbestock** [comment]
How can you tell its species with your antiquated Brille?

**Comment by urapadmin** [comment]
I also feel a pit potty

**Comment by lbestock** [comment]
Wittgenstein this ticket is not. I blame the weather on two continents.

**Comment by urapadmin** [comment]
I can't. It is just some `:potted_plant`

**Comment by luizaogs** [comment]
I had no idea what a pot plant actually looked like before this

**Comment by urapadmin** [comment]
you want some Wittgenstein?

**Comment by urapadmin** [comment]
nobody laughed about my "pit". It's not fun with you guys :o)

**Comment by luizaogs** [comment]
That’s because I read a bit potty because I am blind 🤓

**Comment by urapadmin** [comment]
you, too, need new 👓

**Comment by luizaogs** [comment]
I think I need new eyes

**Comment by lbestock** [comment]
I will leave you blind guys and address myself to broiling an eggplant and An Overview of Egyptian Kingship.

**Comment by urapadmin** [comment]
one day they'll shut down this github repository 🍷

**Comment by urapadmin** [comment]
sounds rather boring 👑

**Comment by urapadmin** [comment]
but then the answer was: No, I don't want any more Wittgenstein.

**Comment by luizaogs** [comment]
that’s because it is, Lutz!

**Comment by urapadmin** [comment]
I am still not past chapter 3 of Götter, Gräber und Gelehrte. After all those years. And that isn't even all that boring.

**Comment by luizaogs** [comment]
try some Jan Assmann

**Comment by urapadmin** [comment]
I did. And did not make it through the introduction,

**Comment by luizaogs** [comment]
the usual reaction

**Comment by urapadmin** [comment]
but in fact because it is very dense and not because it is boring. I just don't have the head space to do it justice.

**Comment by lbestock** [comment]
You cite him. But as "one must know Assmann wrote about cultural memory" only :)

**Comment by urapadmin** [comment]
### Kiosk 0.26.6
- should not happen anymore.

**Comment by luizaogs** [comment]
It doesn't.
